### ISSUE TITLE
LFR Lua Error...

### ISSUE THREAD

**Issue by Milky063** [BigWigsMods/BigWigs#1348]
## Describe the problem

1x BigWigs is missing timers on "LFR" running 265-5c1ee43, tell the devs!
New timer for "373405" at stage 1 with placement 1 and value 35.26.
[string "@BigWigs_Core/BossPrototype.lua"]:436: in function `Disable'
[string "@BigWigs_Core/BossPrototype.lua"]:449: in function `Reboot'
[string "@BigWigs_Core/BossPrototype.lua"]:1060: in function `Wipe'
[string "@BigWigs_Core/BossPrototype.lua"]:1208: in function <BigWigs_Core/BossPrototype.lua:1208>

### What steps will reproduce the problem?

1.  was in LFR killing boss
2.  
3.  

### What version of BigWigs are you using? (Stating 'latest' is not useful)
BigWigs-v265.zip

**Comment by Phiir42** [comment]
Likewise got the error in LFR:
1x BigWigs is missing timers on "LFR" running 265-5c1ee43, tell the devs!
New timer for "373405" at stage 1 with placement 1 and value 35.03.
New timer for "372082" at stage 1 with placement 1 and value 29.94.
[string "@BigWigs_Core/BossPrototype.lua"]:436: in function `?'

**Comment by Xruptor** [comment]
1x BigWigs is missing timers on "LFR" running 265-5c1ee43, tell the devs!
New timer for "373405" at stage 1 with placement 1 and value 34.03.
[string "@BigWigs_Core/BossPrototype.lua"]:436: in function `?'
[string "@BagSync/libs/AceTimer-3.0-17/AceTimer-3.0.lua"]:55: in function <...ceBagSync/libs/AceTimer-3.0/AceTimer-3.0.lua:50>

1x BigWigs is missing timers on "LFR" running 265-5c1ee43, tell the devs!
New timer for "373405" at stage 1 with placement 1 and value 35.60.
[string "@BigWigs_Core/BossPrototype.lua"]:436: in function `?'
[string "@BagSync/libs/AceTimer-3.0-17/AceTimer-3.0.lua"]:55: in function <...ceBagSync/libs/AceTimer-3.0/AceTimer-3.0.lua:50>

**Comment by Foomanshoes** [comment]
I just downloaded the update from 4 days ago, but before that, this was what I got during LFR Raz: 

Message: BigWigs is missing timers on "LFR" running 265-5c1ee43, tell the devs!
New timer for "377467" at stage 3 with placement 1 and value 59.99.
New timer for "377467" at stage 3 with placement 2 and value 57.07.
New timer for "377594" at stage 3 with placement 1 and value 75.03.
New timer for "377594" at stage 3 with placement 2 and value 42.06.
New timer for "385574" at stage 3 with placement 1 and value 31.00.
New timer for "386410" at stage 3 with placement 1 and value 29.94.
New timer for "386410" at stage 3 with placement 2 and value 27.05.
New timer for "386410" at stage 3 with placement 3 and value 30.08.
Time: Sat Feb 18 03:28:38 2023
Count: 1
Stack: BigWigs is missing timers on "LFR" running 265-5c1ee43, tell the devs!
New timer for "377467" at stage 3 with placement 1 and value 59.99.
New timer for "377467" at stage 3 with placement 2 and value 57.07.
New timer for "377594" at stage 3 with placement 1 and value 75.03.
New timer for "377594" at stage 3 with placement 2 and value 42.06.
New timer for "385574" at stage 3 with placement 1 and value 31.00.
New timer for "386410" at stage 3 with placement 1 and value 29.94.
New timer for "386410" at stage 3 with placement 2 and value 27.05.
New timer for "386410" at stage 3 with placement 3 and value 30.08.
[string "@Interface/AddOns/BigWigs_Core/BossPrototype.lua"]:436: in function `Disable'
[string "@Interface/AddOns/BigWigs_Core/BossPrototype.lua"]:449: in function `Reboot'
[string "@Interface/AddOns/BigWigs_Core/BossPrototype.lua"]:1060: in function `Wipe'
[string "@Interface/AddOns/BigWigs_Core/BossPrototype.lua"]:1208: in function <Interface/AddOns/BigWigs_Core/BossPrototype.lua:1208>

Locals: <none>
